docs(agents): add more code annotations (#4300)
langchain-ai/langgraph: docs/docs/agents/multi-agent.md

    print("\n")
```

> 1. Access individual agent's state

Reviewer: ```suggestion
1. Access agent's state
```

> 2. `Command` objects allow you to combine navigation (destination) and state updates (payload), which makes them very useful for implementing handoffs.

Reviewer: ```suggestion
2. The `Command` primitive allows specifying a state update and a node transition as a single operation, making it useful for implementing handoffs.
```

> 3. This is the name of the agent (node) to hand off to.

Reviewer: ```suggestion
3. Name of the agent or node to hand off to.
```

> 4. Take individual agent's messages and **add** them to the parent multi-agent graph's **state**, as part of the handoff. The next agent will see the updated multi-agent graph state.

Reviewer: ```suggestion
4. Take the agent's messages and **add** them to the parent's **state** as part of the handoff. The next agent will see the parent state.
```
